examples/llamaindex-server/README.md

```diff
@@ -0,0 +1,17 @@
+# Run LlamaIndex Server with simple steps
```

Comment:
cool, we can keep this here, even after morving llamaindex-server to the CL repo